README.md

```diff
@@ -269,6 +269,21 @@ export default class App {
 }
 ```
 
+#### Decorators
+
+The `@provide` decorator lets you inject a Redux instance more succinctly:
+
+```js
+import { provide } from 'redux/react';
+
+@provide(redux)
+export default class App {
+  render() {
+    return <CounterApp />;
```

Comment:
i'm going to defer to @gaearon who wrote most of the readme on this one. When reading through the docs the CounterApp is already referenced several times. So holistically I find it easier to learn about the `@provide` feature if the example focuses on the salient details...